header/index.html

```diff
@@ -184,6 +184,25 @@ <h2>Rosangela Oliveira</h2>
   </div>
   <p>Content here.</p>
 </body></textarea>
+    <p>It is possible to change header color at scroll time by calling <code>Header.bind()</code> and then setting up right header attributes. It is recomended to set topClasses value also to class attribute, so there isn't color change animation at launch and also, because page content might show before deviceready event is triggered. Also you need to bind headers, when you call <code>openPage</code> function.</p>
+<textarea class="bind-just-code" mode="text/javascript">
+document.addEventListener("deviceready", onDeviceReady, false);

```

Comment:
This eventListener not necessary put here, just a plugins cordova bind, please remove this bind in here!